src/Illuminate/Support/Collection.php

```diff
@@ -327,6 +327,7 @@ public function get($key, $default = null)
      * Group an associative array by a field or using a callback.
      *
      * @param  callable|string  $groupBy
+     * @param  bool $preserveKeys
```

Comment:
Was asked in a previous comment above to remove the alignment. I tagged you in it.